src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.MaxResponseHeadersLength.cs

```diff
@@ -45,16 +45,30 @@ public void ValidValue_SetGet_Roundtrips(int validValue)
         }
 
         [Fact]
-        [OuterLoop("Uses external server")]
         public async Task SetAfterUse_Throws()
         {
-            using (HttpClientHandler handler = CreateHttpClientHandler())
-            using (HttpClient client = CreateHttpClient(handler))
+            await LoopbackServer.CreateClientAndServerAsync(async uri =>
             {
-                handler.MaxResponseHeadersLength = 1;
-                (await client.GetStreamAsync(Configuration.Http.RemoteEchoServer)).Dispose();
-                Assert.Throws<InvalidOperationException>(() => handler.MaxResponseHeadersLength = 1);
-            }
+                using (HttpClientHandler handler = CreateHttpClientHandler())
+                using (HttpClient client = CreateHttpClient(handler))
+                {
+                    handler.MaxResponseHeadersLength = 1;
+                    (await client.GetStreamAsync(uri.ToString())).Dispose();
```

Comment:
You can pass either a `string` or `System.Uri` type to the HttpClient methods usually. So, you don't need to convert it with ToString().

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.getstreamasync?view=netcore-2.2

Reply:
Done.